src/main/java/com/auth0/client/mgmt/JobsEntity.java

```diff
@@ -50,4 +72,35 @@
         request.setBody(requestBody);
         return request;
     }
+
+    /**
+     * Requests a Users Exports job. A token with scope read:users is needed.
+     * See https://auth0.com/docs/api/management/v2#!/Jobs/post_users_exports
+     * See https://auth0.com/docs/users/guides/bulk-user-exports
+     *
+     * @param connectionId The id of the connection to export the users from
+     * @param filter       the filter to use. Can be null.
+     * @return a Request to execute.
+     */
+    public Request<Job> requestUsersExport(String connectionId, UsersExportFilter filter) {
```

Comment:
Open to name suggestions: `requestUsersExport`, `startUsersExport`, `beginUsersExport`, `postUsersExport`.

Reply:
@jimmyjames @damieng Also, is this supposed to be...? 

a) "userS exportS" 
b) "user exportS"
c) "users export" (the one I chose)

Reply:
I mean I think it should be `exportUsers` ...

Reply:
Updated!